resources/js/i18n/de.json

```diff
@@ -302,12 +308,14 @@
         "csv": {
             "uploader": {
                 "title": "CSV-Uploader",
+                "settings_title": "CSV-Einstellungen",
                 "delimiter": "Trennzeichen",
                 "delimiter_with_info": "Trennzeichen (',', ';', …)",
-                "has_header": "Hat eine Kopfzeile?",
-                "show_linenumbers": "Zeilennummern anzeigen",
+                "has_header": "Kopfzeile",
+                "show_linenumbers": "Zeilennum.",
```

Comment:
I would not use abbrevs in translations

Reply:
Fixed in e1f017ea68742e1c0d73bf36848c7e753edfb3c9